ui/lib/config-ui/addon/components/login-settings/page/list.hbs

```diff
@@ -0,0 +1,38 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<PageHeader as |p|>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      UI Login Rules
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+{{! currently here empty per design, TBD if actions or filters will be added }}
+<Toolbar />
+
+{{#each @loginRules as |rule|}}
+  <div class="list-item-row linked-block-item is-no-underline">
+    <div>
+      <div class="is-grid align-items-center linked-block-title">
+        <Hds::Icon @name="user-check" @size="24" />
+        <div class="has-text-weight-semibold has-left-margin-xs" data-test-rule-name={{rule.name}}>{{rule.name}}</div>
+      </div>
+      <div class="has-top-margin-m" data-test-rule-path={{rule.namespace}}>{{rule.namespace}}</div>
+    </div>
+    <div class="linked-block-popup-menu">
+      <Hds::Dropdown @isInline={{true}} as |dd|>
+        <dd.ToggleIcon @icon="more-horizontal" @hasChevron={{false}} @text="login rules menu" data-test-popup-menu-trigger />
+        <dd.Interactive @route="login-settings.rule" @model={{rule.name}} data-test-view-rule-trigger>
+          View
+        </dd.Interactive>
+        {{#if (has-capability @capabilities "delete" id=rule.name)}}
```

Comment:
👏

Reply:
here capabilities is referenced as an arg, but it's in the component as a service and also in the route model